Makefile

```diff
@@ -4,7 +4,7 @@ build:
 	poetry run ./scripts/build.sh
 
 stop-ajv:
-	npm run stop
+	@PORT=$(PORT) npm run stop
```

Comment:
We could avoid passing any vars manually if we follow what we do in Runner, eg:
```
SCHEMAS_VERSION=`cat .schemas-version`
DESIGN_SYSTEM_VERSION=`cat .design-system-version`
RUNNER_ENV_FILE?=.development.env
SCHEMA_PATH=./schemas/test/en/
```
PORT being assigned similar to "SCHEMAS_VERSION=`cat .schemas-version`"

Reply:
Are we okay with having a file like .schema-version for the Port number soley for the make file command? If a developer did want to change the PORT number they would have to updaet their `.env `as well as the` .port `file.

Reply:
I have made the changes but you may experience a similar issue you had before:
	> source .env && DEBUG=ajv-schema-validator node ajv/app.js &
	sh: .env: No such file or directory